src/pages/MainPage/SSEErrorBoundary.tsx

```diff
@@ -0,0 +1,34 @@
+import { Component, FunctionComponent, ReactElement, ReactNode } from 'react';
+import { BASE_PATH } from '@/generated/base';
+import useSSE, { I_ChangeGameRoomData, SSEErrorType } from '@/hooks/useSSE';
+import storageFactory from '@/utils/storageFactory';
+
+interface SSEErrorBoundary {
```

Comment:
놓치신 것 같아서 말씀드립니다! 😊👍
```suggestion
interface SSEErrorBoundaryProps {
```

Reply:
감사합니다 놓친부분 잘 잡아주셔서 코드가 한층 좋아졌네요 ㅎㅎ